packages/connect-conformance/src/web/transport.ts

```diff
@@ -13,17 +13,17 @@
 // limitations under the License.
 
 import { createRegistry } from "@bufbuild/protobuf";
-import { ClientCompatRequest } from "../gen/connectrpc/conformance/v1/client_compat_pb.js";
+import {
+  createConnectTransport,
+  createGrpcWebTransport,
+} from "@connectrpc/connect-web";
```

Comment:
We're importing this from `@connectrpc/connect-web`, this is confusing. 

Why don't we simply move this to `packages/connect-web/conformance`?
Same for packages/connect-conformance/src/node/transport.ts - I don't see why we shouldn't  simply move it to `packages/connect-node/conformance`.

Reply:
I was just worried if we needed this transport logic in other places as well, but I guess each one is only used in one place.